src/main/java/com/vitalii/twenty/methods/Starter.java

```diff
@@ -28,5 +28,11 @@ public static void main(String[] args) {
 
         //3 приминает 2 инта, а и б, возвращает большее из этих 2х чисел.
         System.out.println(returnMaxOfTwoInts(100500, 300));
+
+        //3.1 приминает 2 инта, а и б, возвращает большее из этих 2х чисел.
+        System.out.println(returnMaxOfTwoIntsWithMath(32, 23));
+
+        //3.2 приминает 2 инта, а и б, возвращает большее из этих 2х чисел.
```

Comment:
Use system.out.println for outing text of task.

Reply:
to be fixed in next pull request